pcs_detection/scripts/data/training_config.json

```diff
@@ -0,0 +1,69 @@
+{
+    "MODE": "TRAIN",
+    "WEIGHT_ID": "new_repo_test",
+    "WEIGHT_DIR": "data/weights",
+
+    "MODEL": "fcn8",
+
+    "CHANNEL": "GREY",
```

Comment:
Could you create a readme in pcs_detection that describes these configs?

Reply:
Added readme with descriptions of each config and an explanation for all config keys